html/semantics/document-metadata/the-base-element/base_target_does_not_affect_iframe_src_navigation.html

```diff
@@ -0,0 +1,10 @@
+<base id="base" target="_blank">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<iframe id="i" src="about:blank"></iframe>
+<script>
+async_test(function(t) {
+  i.src = "data:text/html,This should navigate the iframe";
+  i.onload = () => t.done();
```

Comment:
Are we sure this never fires for the loading of about:blank? (A way around this might be to post a message to the parent instead that marks this as done.)

Reply:
I think Chrome removed the XSS auditor. The reason you had to split that is because of the HTML parser I think, which is roughly uniform across browsers.